docs/en/docs/tutorial/path-params-numeric-validations.md

```diff
@@ -106,7 +106,7 @@ And you can also declare numeric validations:
 * `le`: `l`ess than or `e`qual
 
 !!! info
-    `Query`, `Path` and others you will see later subclasses of a common `Param` class (that you don't need to use).
+    `Query`, `Path`, and others you will see later are subclasses of a common `Param` class (that you don't need to use).
```

Comment:
Why do we need a comma after "Path" when we already have "and".

Reply:
This is a grammatical issue.
To be honest, I don't know how to use commas in this situation.

I don't think the comma in this case should be omitted in either American English or British English. Therefore, I add a comma.

- A, B and C
- **BETTER: A, B, and C**

How is it practically used?

Reply:
Do not get me wrong your PR is perfectly fine by me, I was just curious as a non-native speaker.

I found [this](https://en.wikipedia.org/wiki/Serial_comma). Seems like it is very common for native speakers to put a comma before the last item in a list. They are calling it the Oxford Comma.

Reply:
Feel free to mark the conversation as resolved.

Reply:
Thanks for your comments, @ycd.
I'm a non-native speaker, too.
I learned a lot from you.